code/utils/graphing/plot_neural_behavior.py

```diff
@@ -0,0 +1,38 @@
+"""
+Function to plot the linear regression picture of behavial loss aversion
+and neural loss aversion(-betagain-betaloss) at a single coordinate of standard brain
+"""
```

Comment:
ｉ will correct this one

On Monday, December 14, 2015, Benjamin Hsieh <notifications@github.com>
wrote:

> In code/utils/graphing/plot_neural_behavior.py
> <https://github.com/berkeley-stat159/project-theta/pull/134#discussion_r47470448>
> :
>
> > @@ -0,0 +1,38 @@
> > +"""
> > +Function to plot the linear regression picture of behavial loss aversion
> > +and neural loss aversion(-betagain-betaloss) at a single coordinate of standard brain
> > +"""
>
> need to import numpy as np & from scipy import stats & import
> matplotlib.pyplot as plt
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/berkeley-stat159/project-theta/pull/134/files#r47470448>
> .
>